manifests/4.7/local-volumes.crd.yaml

```diff
@@ -64,7 +64,9 @@ spec:
                         type: string
                       devicePaths:
                         description: 'A list of devices which would be chosen for local storage.
-                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]'
+                        For example - ["/dev/sda", "/dev/sdb", "/dev/disk/by-id/ata-crucial"]
+                        Note: Use the unique /dev/disk/by-id/.. name, if available, to avoid issues
```

Comment:
Really just a nitpick, but I don't think we should have a Note in the CRD. Perhaps something similar to:

> It is strongly recommended to use the unique /dev/disk/by-id/... name if available, as these identifiers persist across node restarts.

Reply:
updated.